src/main/scala/com/ing/wbaa/rokku/proxy/handler/parsers/RequestParser.scala

```diff
@@ -15,14 +17,27 @@ object RequestParser {
 
   case class GetObjectRequestType() extends AWSRequestType
 
-  case class ListObjectRequestType() extends AWSRequestType
+  case class ListObjectsRequestType() extends AWSRequestType
 
   case class PutObjectRequestType() extends ModifyObjectRequestType
 
   case class PostObjectRequestType() extends ModifyObjectRequestType
 
   case class DeleteObjectRequestType() extends ModifyObjectRequestType
 
+  case class ListBucketsRequestType() extends AWSRequestType
+
+  case class BucketOpsRequestType() extends AWSRequestType
+
+  case class ObjectOpsRequestType() extends AWSRequestType
+
+  case class SubFolderOpsRequestType() extends AWSRequestType
+
+  case class MultiDeleteRequestType() extends AWSRequestType
```

Comment:
it is also `ModifyObjectRequestType`
```suggestion
  case class MultiDeleteRequestType() extends ModifyObjectRequestType
```